linode/manager
# M3-5195: RQ-ify user preferences

## Description 📝

Replaces the Preferences Redux store with a React Query hook. Also includes a refactor of `LinodeThemeWrapper` and `PreferenceToggle` to avoid drilling and unnecessary complexity.

## How to test 🧪

Ensure all component/hooks that use user preferences still work as expected:

- `App`
- `DatabaseSettingsDeleteClusterDialog`
- `DeleteKubernetesClusterDialog`
- `DeletionDialog`
- `EnableObjectStorage`
- `LinodeResize`
- `LinodeThemeWrapper`
- `MainContent`
- `MainContentBanner`
- `OrderBy`
- `PreferenceEditor`
- `PreferenceToggle`
- `RebuildFromImage`
- `RebuildFromStackScript`
- `RestoreFromBackupDialog`
- `Settings`
- `TimeRangeSelect`
- `TypeToConfirmDialog`
- `useDismissibleNotifications`
- `useOrder`
- `usePagination`

**How do I run relevant unit or e2e tests?**

`yarn test`
